www/css/artwork.css

```diff
@@ -69,13 +69,13 @@ ol.artwork-list > li img{
 	max-width: 100%;
 }
 
-form[action^="/artworks/"]{
+form:not(.artwork-operate){
```

Comment:
This was some bad CSS. Sorry about that. I tried harder and was able to avoid adding a class. But there are some complicated selectors because there are 4 `<form>`s that have very similar `action` and `method` attributes:

1. The submission form in `artworks/new.php`
2. The edit form in `artworks/edit.php`
3. The Editor Options form in `artworks/get.php`
4. The browse from in `artworks/index.php`

I'm clearly terrible at CSS, including when to decide to give up and add a class to a form or two. Let me know how these selectors look to you.

Reply:
It's OK to use classes if it helps in readability. We want selectors to be basically readable and once you get into multiple nth-child territory it's not as readable anymore. The approach to style for ebooks and web apps is different. The key is not to use special class for literally everything. Classes can be used for clarity but they can also be smartly organized.